puphpet/config.yaml.template

```diff
@@ -79,25 +100,31 @@ apache:
     settings:
         user: www-data
         group: www-data
-        default_vhost: true
+        default_vhost: false
```

Comment:
To more directly answer your question: Static files don't resolve properly from /var/www, so if for some reason mod_rewrite isn't working there via the .htaccess, js, img and css assets won't load from that default vhost.